openmetadata-ui/src/main/resources/ui/src/components/common/atoms/pagination/usePaginationControls.tsx

```diff
@@ -77,23 +131,84 @@ export const usePaginationControls = (config: PaginationControlsConfig) => {
           onChange={(_, page) => config.onPageChange(page)}
         />
 
-        <Button
-          color="secondary"
-          data-testid="next"
-          disabled={config.currentPage >= config.totalPages}
-          endIcon={
-            <ArrowRight
-              style={{ color: theme.palette.allShades?.gray?.[400] }}
-            />
-          }
-          size="small"
-          variant="contained"
-          onClick={() => config.onPageChange(config.currentPage + 1)}>
-          {t('label.next')}
-        </Button>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            gap: 1.5,
+          }}>
+          {config.onPageSizeChange && (
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                gap: 1,
+              }}>
+              <Typography
+                sx={{
+                  fontSize: theme.typography.pxToRem(14),
+                  color: theme.palette.text.secondary,
+                  whiteSpace: 'nowrap',
+                  marginRight: theme.spacing(1),
+                }}>
+                {`${formatNumberWithComma(
+                  displayedRowsRange.start
+                )}-${formatNumberWithComma(displayedRowsRange.end)} ${t(
+                  'label.of'
+                )} ${formatNumberWithComma(config.totalEntities)}`}
+              </Typography>
+              <Select
+                data-testid="rows-per-page-select"
+                size="small"
+                sx={{
+                  '&.MuiOutlinedInput-root .MuiOutlinedInput-input': {
+                    fontSize: `${theme.typography.pxToRem(14)}`,
+                  },
+                  '& .MuiOutlinedInput-input.MuiInputBase-inputSizeSmall': {
+                    padding: '6px 32px 6px 12px !important',
```

Comment:
can you check if we can avoid use of important here?

Reply:
Yes, checked, we need it.